README.md

```diff
@@ -72,75 +72,105 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Support MSC and CPL files in "Run command" results of PowerToys Run Program plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Add support for log2 and log10 in PowerToys Run Calculator plugin. Thanks [@RickLuiken](https://github.com/RickLuiken)!
 
-### Awake
+### General
 
-- Disable instead of hiding "Keep screen on" option.
+- Improve metered network detection in runner. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Update PowerToys logo used by installer. Thanks [@ChaseKnowlden](https://github.com/ChaseKnowlden1)!
+- Add new GPO policies for automatic update downloads, update toast notifications and automatic background update check. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Update copyright year to 2023. Thanks [@ChaseKnowlden](https://github.com/ChaseKnowlden)!
 
 ### FancyZones
 
 - Refactored and improved code quality.
+- Fix crashing on moving window between monitors with Win + arrows. (This was a hotfix for 0.67)
+- Fix issue causing window attributes to not be reset properly. (This was a hotfix for 0.67)
+- Fix issue causing window to not be adjusted when layout is changed. (This was a hotfix for 0.67)
+- Fix issue causing window not to be unsnapped on drag started. (This was a hotfix for 0.67)
+- Fix issue causing layouts not to be applied to new virtual desktops. (This was a hotfix for 0.67)
 
 ### File explorer add-ons
 
-- Fixed escaping HTML-sensitive characters when previewing developer files. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Add Developer files previewer option to set max file size and fix styling issue. [@Aaron-Junker](https://github.com/Aaron-Junker)!
+- Improve Developer files previewer exception handling and error messages display.
+- Fix crash when generating PDF and Gcode files thumbnails. (This was a hotfix for 0.67)
 
-### Image Resizer
+### Hosts file editor
 
-- Improved code quality around a silent crash that was being reported to Microsoft servers.
+- Improve hosts file loading. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Improved duplicate hosts finding. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+
+### Keyboard Manager
+
+- Fix typo in Keyboard Manager Editor. Thanks [@ChristianLW](https://github.com/ChristianLW)!
+
+### Mouse Utils
+
+- Resolve grammatical error in Mouse Highlighter description. Thanks [@WordlessSafe1](https://github.com/WordlessSafe1)!
+- New utility: Mouse Jump allows to quickly move the mouse pointer long distances on single or across screens. Thanks [@mikeclayton](https://github.com/mikeclayton)!
+
+### Paste as Plain Text
+
+- New utility: Paste as Plain Text allows pasting the text contents of your clipboard without formatting. Note: the formatted text in the clipboard is replaced with the unformatted text. Thanks [@carlos-zamora](https://github.com/carlos-zamora)!
```

Comment:
I would put this on top

Reply:
what do you mean? It is listeďin highlights

Reply:
Fair point